site/content/docs/credentials.en.md

```diff
@@ -1,4 +1,6 @@
-This section explains our recommendations around credentials to provide the best experience with the AWS Copilot CLI.
+AWS Copilot uses AWS credentials to access AWS API, store and look up an [application's metadata](concepts/applications.en.md), and deploy and operate application's workloads.
+
+You can learn more on how to configure AWS credentials at [Quick configuration with `aws configure`](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config).
```

Comment:
This is super helpful for first-time users. I remember being confused by this when I first started! Thank you.